signup/signup.css

```
@@ -0,0 +1,239 @@
+:root {
+  --primary-0: #e6f2ff;
+  --primary-50: #cfe5ff; /* background */
+  --primary-100: #3692ff; /* default */
+  --primary-200: #1967d6; /* hover */
+  --primary-300: #1251aa; /* active, focus */
+  --secondary-50: #f9fafb;
+  --secondary-100: #f3f4f6; /*input bg*/
+  --secondary-200: #e5e7eb;
+  --secondary-300: #fcfcfc; /* card bg, bottom banner bg */
+  --secondary-350: #dfdfdf;
+  --secondary-400: #9ca3af; /* inactive */
+  --secondary-500: #6b7280;
+  --secondary-600: #4b5563;
+  --secondary-700: #374151;
+  --secondary-800: #1f2937;
+  --secondary-900: #111827; /* footer bg*/
+  --error: #f74747;
+}
+* {
+  box-sizing: border-box;
+  flex-wrap: wrap;
+  font-size: 10px;
+  font-family: 'Pretendard';
+}
```

❗️ 수정요청
해당 부분이 base.css와 동일하니 index.css에서 하신 것처럼 import해오시면 더 좋을 것 같아요!

```suggestion
@import url(base.css);
```